pkg/apis/core/validation/controllerdeployment_test.go

```diff
@@ -79,7 +79,20 @@ var _ = Describe("#ValidateControllerDeployment", func() {
 		}))))
 	})
 
-	Context("helm type", func() {
+	It("helm type: should not allow both rawChart and OCIRepository", func() {
```

Comment:
there are still contexts for `helm type with rawChart` and `helm type with ociRepository` and I wanted to avoid deeply nesting stuff, but I guess its not too deep yet compared to other tests